infoschema/tables.go

```diff
@@ -745,6 +749,24 @@ var tableClusterLoadCols = []columnInfo{
 	{"LOAD_VALUE", mysql.TypeVarchar, 128, 0, nil, nil},
 }
 
+var tableTiDBClusterHardwareCols = []columnInfo{
+	{"TYPE", mysql.TypeVarchar, 64, 0, nil, nil},
+	{"ADDRESS", mysql.TypeVarchar, 64, 0, nil, nil},
+	{"DEVICE_TYPE", mysql.TypeVarchar, 64, 0, nil, nil},
+	{"DEVICE_NAME", mysql.TypeVarchar, 64, 0, nil, nil},
```

Comment:
What's the diffrence between DEVICE_TYPE and DEVICE_NAME ?

Reply:
eg:
```sql
decive_type: disk
device_name: disk1,disk2...
```